src/componentsEventPage/Header.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components/macro'
+import CategoryFilter from './CategoryFilter'
+
+export default function Header({ onSelectFilter }) {
+    return (
+        <header>
+            <ButtonContainer>
```

Comment:
I think you don't need this `div` at this point

Reply:
yes, I need to integrate my design :D